src/register-org/register-org.module.ts

```diff
@@ -10,13 +11,15 @@ import * as fromServices from './services';
 @NgModule({
   imports: [
     CommonModule,
+    ExuiCommonLibModule,
     HttpClientModule,
+    ExuiCommonLibModule,
```

Comment:
ExuiCommonLibModule imported twice.

Reply:
yes, think was not picked up in merge conflicts